src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -153,6 +180,43 @@
           [Button {:text (build-text "Exception" :exception) :state state :filter :exception}]
           [:div {:style {:clear "both"}}]]))}))
 
+(react/defc EntityFilterButtons
```

Comment:
We should really factor this out into a common component.  I realize we're trying to close out a sprint, so we can do this later.